modules/porous_flow/test/tests/hysteresis/except14.i

```diff
@@ -0,0 +1,145 @@
+# Exception: attempting to use PorousFlow2PhaseHysPP in a 1-phase situation
```

Comment:
Again really minor, but you could probably get rid of things like the post processors and aux kernels in these exception tests?

Reply:
agreed.  done